microsoft-edge/webview2/how-to/playwright.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Use Playwright to automate and test in WebView2
+description: Using Playwright to automate and test webpages in a WebView2 control.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: how-to
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 06/06/2023
+---
+# Use Playwright to automate and test in WebView2
+
+Use Playwright to automate and test webpages in a WebView2 control.  See [WebView2](https://github.com/microsoft/playwright/blob/80c593e6396cc0d9a41e9cdf594f2b7aa303bed9/docs/src/webview2.md) at GitHub.
```

Comment:
Is there a cleaner URL possible -- not a commit-specific link?
Which version of page to link to -- why is Playwright docs version shorter?

Reply:
outdated